web/components/templates/requests/components/chatComponent/ChatMessageTopBar.tsx

```diff
@@ -32,8 +32,53 @@ interface ChatMessageTopBarProps {
   deleteMessage: (_index: number) => void;
   popoverOpen: boolean;
   setPopoverOpen: (_open: boolean) => void;
+  onAddText?: () => void;
+  onAddImage?: () => void;
 }
 
+const ROLE_OPTIONS = [
+  { value: "user", label: "User" },
+  { value: "assistant", label: "Assistant" },
+  { value: "system", label: "System" },
+  { value: "tool", label: "Tool" },
+  { value: "image", label: "Image" },
```

Comment:
probably dont want images as a role? i just added it at the time to test out the starting code - we can get rid of this (and all related code) if the only purpose of it is to add a new image and not show anything